pydantic/main.py

```diff
@@ -678,7 +679,7 @@ def _get_value(
         value_exclude = ValueItems(v, exclude) if exclude else None
         value_include = ValueItems(v, include) if include else None
 
-        if isinstance(v, dict):
+        if isinstance(v, collections.abc.Mapping):
```

Comment:
It should be `typing.Mapping`.
But I'm not sure it's expected.
If I have
```py
    class Model(BaseModel):
        a: MyCustomMap[str, int]
```
I expect my value in the dict representation to be a instance of `MyCustomMap`, not a dict